src/styles/blocks/card.scss

```diff
@@ -0,0 +1,105 @@
+@import "../utils/variables";
+
+.card {
+  width: $width-card;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  border: 1px solid $border-color-card;
+  transition: all 300ms ease-in-out;
+
+  &__name {
+    margin-bottom: 4px;
+    color: $dark-blue-color-card;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &:hover &__name {
+    color: $hover-color-card;
+  }
+
+  &__img {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__code {
+    color: $grey-color-card;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+    margin-bottom: 16px;
+  }
+
+  &__container-stars {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__reviews {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    color: $dark-blue-color-card;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    color: $grey-color-card;
+    font-family: Roboto, sans-serif;
```

Comment:
![image](https://github.com/mate-academy/layout_catalog/assets/54860641/5fb7472d-2e8e-4b44-9bcf-b3d4d89456f2)

Reply:
Not fixed